docs/reference/configuration/storage-config.md

```diff
@@ -104,7 +94,7 @@ In this configuration, all KVStore C APIs will be mapped to the TDBStore in the
 
 Below is the `TDB_INTERNAL` configuration in `mbed_lib.json`:
 
-```
+``` json
```

Comment:
This appears to work on GitHub OK. Do we know if our docs engine needs a space after the triple backticks or if it can't cope with a space there?

Reply:
Noticed the same when reviewing the other PRs on this topic. What about just not have any space at all?

Reply:
I can't find any documentation that says there shouldn't be a space, although all the snippets I've seen don't add one.

Reply:
I don't feel strongly either way, so I'll remove the space just in case.